ports/psoc6/mpconfigport.h

```diff
@@ -96,7 +88,7 @@
 
 #define MICROPY_PY_URE_MATCH_GROUPS             (1)
 #define MICROPY_PY_URE_MATCH_SPAN_START_END     (1)
-#define MICROPY_PY_UCRYPTOLIB                   (0)
+// #define MICROPY_PY_UCRYPTOLIB                   (0)
```

Comment:
Commented here and enabled in line 165. Remove this line?

Reply:
leftover, removing it